src/inputs/input.jsx

```diff
@@ -151,40 +201,36 @@ class Input extends React.PureComponent {
                 }
 
                 this.inputTimer = setTimeout(() => {
-                    if (_.isEmpty(newValue)) {
+                    if (isEmpty(newValue)) {
                         if (required) {
-                            newValue = _.isNumber(min) ? min : (_.isNumber(max)? max : 0);
+                            if (isNumber(min)) {
+                                newValue = min;
+                            } else if (isNumber(max)) {
+                                newValue = max;
+                            } else {
+                                newValue = 0;
+                            }
```

Comment:
Nice.  Probably to appease ESLINT "no-nested-ternary" rule I imagine.  But I happen to agree ... it does improve readability.

Reply:
I totally agree. the nested ternary methods are horrible to read.

Reply:
Resolving this thread to reduce noise on PR.